app/service/govuk_site_lookup_service.rb

```diff
@@ -0,0 +1,29 @@
+class GovukSiteLookupService
+  def govuk_page?(slug)
+    public_url_info.key?(slug)
+  end
+
+  def page_link(slug)
+    public_url_info[slug][:link]
+  end
+
+  def page_title(slug)
+    public_url_info[slug][:title]
+  end
+
+  def public_url_info
+    Rails.cache.fetch("imminence_public_url_info", expires_in: 5.minutes) do
+      content_store_api = GdsApi.content_store
+      place_pages = GdsApi.search.search({ filter_format: "place", count: 200, fields: "title,link,content_id" })
```

Comment:
Do we need to pull the `content_id` too? Can't see it being used

Reply:
Nope. removed.